tests/50federation/00prepare.pl

```diff
@@ -11,51 +11,57 @@
 
 my $DIR = dirname( __FILE__ );
 
-push our @EXPORT, qw( INBOUND_SERVER OUTBOUND_CLIENT );
+push our @EXPORT, qw( INBOUND_SERVER OUTBOUND_CLIENT create_federation_server );
+
+sub create_federation_server {
+   my $server = SyTest::Federation::Server->new;
+   $loop->add( $server );
+
+   require IO::Async::SSL;
```

Comment:
mmkay. I was just cargo-culting the original (https://github.com/matrix-org/sytest/pull/353/files#diff-ae1f9e915c208806589880fe9bd6e5f5L21). Moved anyway.